Delete cascade on composition relations (#7185)
qgis/QGIS-Documentation: docs/user_manual/working_with_vector/attribute_table.rst

  and the child layer. The default :guilabel:`Association` type means that
  the parent layer is *simply* linked to the child one while the
  :guilabel:`Composition` type allows you to duplicate also the child features
  when duplicating the parent ones and on deleting a feature the children are

>   deleted as well, resulting in cascade over all levels (means childrenchildren
>   and childrenchildrenchildren are deleted as well).

Reviewer: Can we instead go with something translation friendlier as 
```suggestion
  deleted as well, resulting in cascade over all levels (means children of children
  of... are deleted as well).
```
though I wonder if delete cascade is not meaningful _per se_

Author: I think both options would be okay, dont have a strong opinion if we go only with cascade or with the explanation

Reviewer: OK. Let's go with the explained option then.

  and the child layer. The default :guilabel:`Association` type means that
  the parent layer is *simply* linked to the child one while the
  :guilabel:`Composition` type allows you to duplicate also the child features
  when duplicating the parent ones and on deleting a feature the children are

>   deleted as well, resulting in cascade over all levels (means childrenchildren
>   and childrenchildrenchildren are deleted as well).

Reviewer: ```suggestion
  deleted as well, resulting in cascade over all levels (means children of children
  of... are deleted as well).
```
